pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/ModularLoadManagerStrategy.java

```diff
@@ -49,12 +49,18 @@ Optional<String> selectBroker(Set<String> candidates, BundleData bundleToAssign,
     /**
      * Create a placement strategy using the configuration.
      *
+     * @param conf ServiceConfiguration to use.
      * @return A placement strategy from the given configurations.
      */
-    static ModularLoadManagerStrategy create() {
+    static ModularLoadManagerStrategy create(final ServiceConfiguration conf) {
         try {
-            // Only one strategy at the moment.
-            return new LeastLongTermMessageRate();
+            Class<?> loadAssignmentClass = Class.forName(conf.getLoadBalancerLoadPlacementStrategy());
+            Object loadAssignmentInstance = loadAssignmentClass.getDeclaredConstructor().newInstance();
```

Comment:
please take a look to other parts in Pulsar code, where we load classes with reflection, we should have some common code